server/src/main/scala/org/http4s/netty/server/websocket/ZeroCopyBinaryText.scala

```diff
@@ -0,0 +1,17 @@
+package org.http4s.netty.server.websocket
+
+import org.http4s.websocket.WebSocketFrame
+import scodec.bits.ByteVector
+
+import java.nio.charset.StandardCharsets
+
+final case class ZeroCopyBinaryText(content: Array[Byte], last: Boolean) extends WebSocketFrame {
```

Comment:
could this maybe take a ByteVector instead, thereby having it "mostly" immutable, we could have an apply method for raw byte arrays that signals the unsafety of that.
That constructor could just wrap it in a view as the current `data` impl has